### PR TITLE
add fill_missing

### PR DESCRIPTION
**Pull request by johnduva**
WIP
added fill_missing=True to relevant lines in Neuron_trace.py and session.py

### CODE REVIEW

File: brainlit/utils/Neuron_trace.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ class NeuronTrace:
             If s3 is provided, specifies if it should be rounded, default True
         read_offset: bool
             If swc is provided, whether offset should be read from file, default False
+        fill_missing: bool
+            Fill missing values with 0s
```
Comment (tathey1) [inline brainlit/utils/Neuron_trace.py:32]: I think a little more specificity is warranted here. As you know NeuronTrace, and NeuroglancerSession do a lot of things. So, what type of "values" are you referring to? trace data? image data? during which methods within the class will 0s possibly be added?

**Review by tathey1 — CHANGES_REQUESTED** [review]
Also, I think worth merging with the current version of develop, since the Algorithms section of your docs page here is not rendering.

File: brainlit/utils/Neuron_trace.py (hunk 1 above)
Comment (johnduva) [reply brainlit/utils/Neuron_trace.py:32]: How about: 
`Always passes directly into 'CloudVolume()' function to fill missing skeleton values with 0s`

Comment (johnduva) [reply brainlit/utils/Neuron_trace.py:32]: I don't think I understand the above comment about merging with develop. Isn't the point of the pull request to merge these changes into develop?

Comment (tathey1) [reply brainlit/utils/Neuron_trace.py:32]: > How about:
> `Always passes directly into 'CloudVolume()' function to fill missing skeleton values with 0s`

Isn't it filling image values? not skeleton values?

Comment (tathey1) [reply brainlit/utils/Neuron_trace.py:32]: > I don't think I understand the above comment about merging with develop. Isn't the point of the pull request to merge these changes into develop?

The point is your docs aren't building completely correctly, e.g. the algorithms section is blank [here](https://deploy-preview-265--brainlit.netlify.app/reference/algs.html), whereas it should not be blank see [here](http://brainlit.neurodata.io/reference/algs.html). If you merge your branch with develop, then this should be fixed.

Comment (johnduva) [reply brainlit/utils/Neuron_trace.py:32]: > > How about:
> > `Always passes directly into 'CloudVolume()' function to fill missing skeleton values with 0s`
> 
> Isn't it filling image values? not skeleton values?

Within Neuron_trace.py, it is always implemented in the form:
```
cv = CloudVolume(path, mip=mip, fill_missing=fill_missing)
skeleton = cv.skeleton.get(seg_id)
```
However, in session.py, it is usually in regard to segments, which I believe are always image values. So the docstring in session.py would be different from that of Neuron_trace.py.

Comment (johnduva) [reply brainlit/utils/Neuron_trace.py:32]: > > I don't think I understand the above comment about merging with develop. Isn't the point of the pull request to merge these changes into develop?
> 
> The point is your docs aren't building completely correctly, e.g. the algorithms section is blank [here](https://deploy-preview-265--brainlit.netlify.app/reference/algs.html), whereas it should not be blank see [here](http://brainlit.neurodata.io/reference/algs.html). If you merge your branch with develop, then this should be fixed.

Interesting. I'm not sure why that is, all I've done is edit the two files we've been talking about. Okay, so is there any action item to do outside of this PR being accepted after I finish the docstrings?

Comment (tathey1) [reply brainlit/utils/Neuron_trace.py:32]: well you should make the docs right.

```
git checkout develop
git pull
git checkout johnduva-fill_missing
git merge develop
```

Comment (johnduva) [reply brainlit/utils/Neuron_trace.py]: > well you should make the docs right.
> 
> ```
> git checkout develop
> git pull
> git checkout johnduva-fill_missing
> git merge develop
> ```

Got it. It seems to be working now.

**Review by tathey1 — APPROVED** [review]
comments addressed

**Pull request merged**